services/blockchain-app-registry/shared/metadata.js

```diff
@@ -0,0 +1,85 @@
+/*
+* LiskHQ/lisk-service
+* Copyright © 2022 Lisk Foundation
+*
+* See the LICENSE file at the top-level directory of this distribution
+* for licensing information.
+*
+* Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+* no part of this software, including this file, may be copied, modified,
+* propagated, or distributed except according to the terms contained in the
+* LICENSE file.
+*
+* Removal or modification of this copyright notice is prohibited.
+*
+*/
+const { MySQL: { getTableInstance } } = require('lisk-service-framework');
+
+const config = require('../config');
+
+const MYSQL_ENDPOINT = config.endpoints.mysql;
+
+const applicationsIndexSchema = require('./database/schema/applications');
+
+const getApplicationsIndex = () => getTableInstance(
+	applicationsIndexSchema.tableName,
+	applicationsIndexSchema,
+	MYSQL_ENDPOINT,
+);
+
+const getBlockchainAppsMetaList = async (params) => {
+	const applicationsDB = await getApplicationsIndex();
+
+	const blockchainAppsMetaList = {
+		data: [],
+		meta: {},
+	};
+
+	if (params.search) {
+		const { search, ...remParams } = params;
+		params = remParams;
+
+		params.search = {
+			property: 'name',
+			pattern: search,
+		};
+	}
+
+	const [{ count: total }] = await applicationsDB.rawQuery('SELECT COUNT(DISTINCT(name)) as count from applications');
+
+	const defaultApps = await applicationsDB.find(
+		{ ...params, isDefault: true },
```

Comment:
Please adjust the limit to be `params.limit * <num_of_networks_supported>`.

SUPPORTED_NETWORKS can contain a list of networks such as `mainnet`, `testnent`, `betanet`, `alphanet`, `devnet`.